theatre/core/src/sheetObjects/TheatreSheetObject.ts

```diff
@@ -32,8 +34,11 @@ export interface ISheetObject<Props extends IShorthandCompoundProps = {}> {
    * const obj = sheet.object("obj", {x: 0})
    * console.log(obj.value.x) // prints 0 or the current numeric value
    * ```
+   *
+   * Question: What should we do with our type, here, if the user actually changes the
```

Comment:
Right now the user can't, but we'll probably enable that functionality later (`object.overrideConfig()`). We should just educate the user that they can't rely on static types to know the type of `object.value`.

Reply:
Got it. Adding comment to this effect